test/download-transformer.test.js

```diff
@@ -0,0 +1,198 @@
+                        it('should return govukSummaryList instruction without the actions change links', () => {
+                            const result = qTransformer.transform({
+                                schemaKey: 'p--check-your-answers',
+                                schema: escapeSchemaContent(schemaArrayOfThemes),
+                                uiSchema: {
+                                    'p--check-your-answers': {
+                                        options: {
+                                            pageContext: 'summary'
+                                        }
+                                    }
+                                }
+                            });
+
+                            const expected = {
+                                hasErrors: false,
+                                content: schemaArrayOfThemesContent,
+                                pageTitle: 'Check your answers'
+                            };
+
+                            expect(removeIndentation(result)).toEqual(removeIndentation(expected));
+                        });
+                    });
+                });
+
+                describe('And the summaryStructure key is present in summaryInfo', () => {
+                    it('should return a govukSummaryList instruction', () => {
+                        const result = qTransformer.transform(schemaComposite);
+
+                        const expected = {
+                            componentName: 'summary',
+                            content: schemaCompositeContent,
+                            dependencies: [
+                                '{% from "summary-list/macro.njk" import govukSummaryList %}'
+                            ],
+                            id: 'p--check-your-answers'
+                        };
+
+                        expect(removeIndentation(result)).toEqual(removeIndentation(expected));
+                    });
+
+                    it('should return a govukSummaryList instructions under the correct headings with no change links', () => {
+                        const result = qTransformer.transform(schemaCompositeAndDate);
+
+                        const expected = {
+                            componentName: 'summary',
+                            content: schemaCompositeAndDateContent,
+                            dependencies: [
+                                '{% from "summary-list/macro.njk" import govukSummaryList %}'
+                            ],
+                            id: 'p--check-your-answers'
+                        };
+
+                        expect(removeIndentation(result)).toEqual(removeIndentation(expected));
+                    });
+                });
+
+                it('should return a govukSummaryList in the order defined by the template and append answers which are supplied in the body but not defined in the template with no change links', () => {
```

Comment:
any way you could rewrite this test name?

Reply:
This is a modification of this [test ](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/b02d10501058f336c654882d484b1611954b3b81/test/q-transformer.test.js#L1696)for the download test